### PR TITLE
Add jenkins agent images

### PR DESCRIPTION
**Pull request by fwilhe**

### CODE REVIEW

File: jenkins-agent-k8s/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM jenkins/jnlp-slave:3.23-1-alpine
```
Comment (rkamath3) [inline jenkins-agent-k8s/Dockerfile]: `3.27-1 ` is the latest version

File: jenkins-agent-k8s/Readme.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Jenkins agent k8s
+
+This is a docker version of Jenkins JNLP agent  which is an extension to [jenkinsci/jnlp-slave](https://hub.docker.com/r/jenkins/jnlp-slave/).
+
+The `ppiper/jenkins-master` runs with a user id `1000`.
+Hence, a user `piper` with uid `1000` has been added to the JNLP agent as well avoid the possible issue with the access rights of the files that needs to be accessed by both master and the agent.
```
Comment (rkamath3) [inline jenkins-agent-k8s/Readme.md]: requires rephrasing

File: jenkins-agent/Readme.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+This image is published to Docker Hub and can be pulled via the command
+
+```
+docker pull ppiper/jenkins-agent
+```
+
+## Build
+
+To build this image locally, open a terminal in the directory of the Dockerfile an run
```
Comment (rkamath3) [inline jenkins-agent/Readme.md]: an ->and

File: jenkins-agent-k8s/Readme.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+This image is published to Docker Hub and can be pulled via the command
+
+```
+docker pull ppiper/jenkins-agent-k8s
+```
+
+## Build
+
+To build this image locally, open a terminal in the directory of the Dockerfile an run
```
Comment (rkamath3) [inline jenkins-agent-k8s/Readme.md]: an -> and

File: jenkins-agent-k8s/Dockerfile (hunk 1 above)
Comment (fwilhe) [reply jenkins-agent-k8s/Dockerfile]: Since you did the upstream image, I would appreciate if you just apply the fixes to this branch. Thanks :)

**Comment by rkamath3** [comment]
I agree that we need a readme file for jenkins-master. But, let's create it as new PR, not part of this one.

**Comment by rkamath3** [comment]
Fixes #28

**Pull request merged**